bagua-core-internal/src/datatypes/mod.rs

```diff
@@ -983,6 +1072,7 @@ impl BaguaBucket {
                 tensors: tensors.iter().map(|x| (**x).clone()).collect(),
                 comm_ops: vec![],
                 dtype: tensors.first().unwrap().inner.read().raw.dtype().clone(),
+                states: Default::default(),
```

Comment:
don't do this, make operator specific logic local to the operator itself.

you can do stateful operator.